opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -444,6 +431,7 @@ def start_span(
             parent: The span's parent. Defaults to the current span.
             kind: The span's kind (relationship to parent). Note that is
                 meaningful even if there is no parent.
+            links: The span's links, to be used in sampling decisions
```

Comment:
I don't think we should call out sampling here. This is merely a detail, mostly the links will be used to, well, have the span linked to other spans, not to influence sampling.